src/software/gui/shared/parameters_spinbox.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+#include <QtWidgets/QSpinBox>
+#include <memory>
+
+#include "software/parameter/parameter.h"
+
+/**
+ * Helper function for setting the value of the spinbox
+ *
+ * @param spin_box
+ * @param parameter
```

Comment:
could you expand on what each of these params are supposed to contain?

i.e

~~~
@param spin_box Your desc of what spin_box is here
@param parameter Your desc of what the paramater is here
~~~

Reply:
Added params descriptions.
How do I go about running the visualizer and checking the spin boxes?

Reply:
`bazel run //software:full_system -- --backend=WifiBackend --interface=wlp3s0` run this command in your terminal, except for `wlp3s0` put your network interface which you can find with `ifconfig`

if you change the spin boxes, you should see logs in the terminal that you ran the visualizer

Reply:
Ran the visualizer and spin boxes are working.